src/QirRuntime/lib/Tracer/tracer-qis.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright (c) // NOLINT{} Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include <assert.h>

```

Comment:
`#include <cassert>` ?
The C++ wrappers (like `<cassert>`) put some of the C's identifiers into the `std::` namespace.
If the C's headers (like `<assert.h>`) are included directly to C++, then this results in the identifiers outside of `std::` namespace, which results in a non-portable C++ code.
[Example](https://en.cppreference.com/mwiki/index.php?title=Special%3ASearch&search=atan2).